src/QMCWaveFunctions/SPOSet.h

```diff
@@ -24,6 +24,7 @@
 #include "Particle/ParticleSet.h"
 #include "Particle/VirtualParticleSet.h"
 #include "QMCWaveFunctions/OrbitalSetTraits.h"
+#include "OptimizableObject.h"
```

Comment:
Doesn't seem needed.

Reply:
Need this to propagate typedef. Will connect SPOSet to OptimizableObject later.